Gemfile.lock

```diff
@@ -19,4 +23,4 @@ DEPENDENCIES
   sinatra
 
 BUNDLED WITH
-   1.11.2
+   2.1.4
```

Comment:
This and the deps is a lot of upgrades. Is everything working after these changes?